Gems/ROS2/Code/Source/Manipulation/JointsManipulationComponent.cpp

```diff
@@ -449,4 +470,31 @@ namespace ROS2
         }
         MoveToSetPositions(deltaTime);
     }
+
+    void JointsManipulationComponent::ProcessPositionControlMessage(const std_msgs::msg::Float64MultiArray& message)
+    {
+        if (message.data.size() != m_initialPositions.size())
```

Comment:
I am not fan of this design. 
Let imagine that we have an articulated contraption with four legs. 
And only part of the system is controlled by control subscription.
In that situation  :
- we can have multiple topics for different segments of our contraption.
- we will have different number of initial joints position than subscription.
IHMO we are creating here a lot inflexibility.
Consider:
moving subscription to separate component with own topic and joint name configuration. I think it would be more flexible.